extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+
+### STAC Item Properties
+
+#### CARD4L
+
+| Field Name                         | Data Type | Description                                                  | Req.  |
+| ---------------------------------- | --------- | ------------------------------------------------------------ | ----- |
+| card4l:specification               | string    | **REQUIRED.** The CARD4L specification implemented, either `SR` (Optical, Surface Reflectance) or `ST` (Optical, Surface Temperature). | *n/a* |
+| card4l:specification_version       | string    | **REQUIRED.** The CARD4L specification version. Currently always `5.0`. | *n/a* |
+| card4l:northern_geometric_accuracy | number    | An estimate of the northern geometric accuracy in meters.    | 1.8   |
+| card4l:eastern_geometric_accuracy  | number    | An estimate of the eastern geometric accuracy in meters.     | 1.8   |
+
+Note that all these fields here are aligned with the CARD4L SAR extension.
+
+#### Common Metadata
+
+| Field Name     | Description                                                  | Req.  |
+| -------------- | ------------------------------------------------------------ | ----- |
+| license        | Recommended to be specified in a STAC Collection.            | *n/a* |
+| datetime       | **REQUIRED.** The time of the acquisition, usually the central timestamp between `start_datetime` and `end_datetime`. | 1.3   |
+| start_datetime | Start time of the acquisition.                               | 1.3   |
+| end_datetime   | End time of the acquisition.                                 | 1.3   |
+| instruments    | **REQUIRED.**                                                | 1.9   |
+| constellation  | Constellation name in lower-case. Only if part of a constellation, e.g. `sentinel-2` for Sentinel 2A and 2B. Can often be derived from `platform`. | (1.9) |
+| platform       | Platform name in lower-case. Use a specific name such as `sentinel-2a` if part of constellation. MUST NOT duplicate `constellation`. | (1.9) |
+
+#### EO (Electro-Optical)
+
+| Field Name     | Description                                                  | Req.   |
+| -------------- | ------------------------------------------------------------ | ------ |
+| eo:cloud_cover |                                                              | 1.17   |
+| eo:bands       | You may include all bands specified in the assets again as an overview. | (1.10) |
+
+#### Processing
+
+| Field Name          | Description                                                  | Req.        |
+| ------------------- | ------------------------------------------------------------ | ----------- |
+| processing:facility |                                                              | 1.15        |
+| processing:software | String likely needs to be split into software name and version number. | 1.13 / 1.15 |
+| processing:lineage  | Additional processing information and parameters, e.g. a description of the processing chain. | 1.15        |
```

Comment:
These tables don't include `Type` like in the main spec and in extensions. Is it a string? Is there any guidance on what to put here or just a prose description of the processing?

Reply:
It's 1:1 like in the main description. I have not included data type and original description to avoid duplication

Reply:
I've added the following part at the top of the extension:

> You have to read the STAC extensions in combination with this extension as this extension just provides the mapping between the STAC fields and the CARD4L requirements, but this extension doesn't give information on the data type or an actual detailed description about the fields.